### ISSUE TITLE
Cannot build with latest libsuperderpy

### ISSUE THREAD

**Issue by suve** [dos1/cinf#1]
I'm trying to build `cinf` with the latest `libsuperderpy` and I get the following errors:
```
src/gamestates/menu.c:31:6: error: conflicting types for 'Gamestate_Logic'
note: previous declaration of 'Gamestate_Logic' was here
    /usr/include/libsuperderpy/libsuperderpy.h:156:6: 
    void Gamestate_Logic(struct Game* game, struct GamestateResources* data);
```
and so on, for all the `Gamestate_` functions.

**Comment by dos1** [comment]
Yeah, there were some minor API changes going on recently and I'm gradually updating older games to it.

**Comment by suve** [comment]
No worry. I'm currently trying to package some of the games for Fedora, so you can expect a lot of bug reports in the next few weeks. Or not, if everything turns out to work okay.  **:)**

**Comment by dos1** [comment]
Fixed by 7a182a151855260c0bb66afe66f4fb257f100332

**Comment by dos1** [comment]
FYI: For packaging libsuperderpy games, I recommend using "-DLIBSUPERDERPY_STATIC=on" cmake switch. libsuperderpy doesn't have stable API yet and there's no proper versioning so far, so ABI isn't stable as well :D
